mentoris/views.py

```diff
@@ -136,3 +136,25 @@ def request_translation(request, user_id):
         "notifications@kontinua.org",
         [email],
     )
+
+def main(request, volume_id = 1):
```

Comment:
I'm not sure about this spacing. Please run the file with the Python Formatter we have to make sure the document is formatted correctly.